### PR TITLE
GH-1884 AutoComplete with interactions.

### PR DESCRIPTION
**Pull request by mounicarachakonda**
Fixes GH-1884

Signed off by <mounica.rachakonda@lexisnexis.com>

### CODE REVIEW

**Comment by jpellock** [comment]
Signed off by @jpellock

File: src/other/AutoCompleteText.js [hunk 1]
```diff
@@ -71,8 +71,23 @@
                     var re = new RegExp("(" + search.split(' ').join("|") + ")", "gi");
                     return '<div class="autocomplete-suggestion" data-val="' + item.value + '" data-row="' + item.origRow + '">' + item.value.replace(re, "<b>$1</b>") + '</div>';
                 },
-                onSelect: function (e, term, item) {
-                    context.click(item.getAttribute("data-row"), context.valueColumn(), true);
```
Comment (GordonSmith) [inline src/other/AutoCompleteText.js]: Would the following change have fixed it:
```context.click(context.RowToObj(item.getAttribute("data-row")), context.valueColumn(), true);```

Comment (mounicarachakonda) [inline src/other/AutoCompleteText.js]: @GordonSmith , when rowToObj is added the result is dividing and placing each letter in the field 
Click:  {"Subject":"G","Year 1":"e","Year 2":"o","Year 3":"g","Year 4":"r"}, Subject, true.

Comment (mounicarachakonda) [inline src/other/AutoCompleteText.js]: @GordonSmith  Please review.

**Comment by mounicarachakonda** [comment]
@GordonSmith  Please review.

**Comment by GordonSmith** [comment]
Closing in favor of:  https://github.com/hpcc-systems/Visualization/pull/1910

**Pull request closed without merging**